### PR TITLE
HL-334

### PR DESCRIPTION
**Pull request by JasminFarhi**
[Link to ticket](https://dltx.atlassian.net/jira/software/projects/HL/boards/82?selectedIssue=HL-334)

- Adjust toggle colour and styling for filter
Screen Shot 2022-12-13 at 1 12 17 pm

### CODE REVIEW

File: src/components/Dashboard/DashboardFilterGroup.tsx [hunk 1]
```diff
@@ -27,8 +27,9 @@ export const DashboardFilterGroup: React.FC<Props> = ({
           onChange(key);
         }}
         className={classNames("bg-white rounded px-2 shadow w-28", {
-          "bg-indigo-600 hover:bg-indigo-700 text-white": key === currentOption,
-          "disabled:opacity-75 hover:bg-indigo-700": key !== currentOption
+          "bg-indigo-600  text-white hover:bg-indigo-700":
```
Comment (brettvniekerk) [inline src/components/Dashboard/DashboardFilterGroup.tsx:30]: remove double space between 600 and text pls

File: src/components/Dashboard/DashboardFilterGroup.tsx [hunk 2]
```diff
@@ -37,3 +38,5 @@ export const DashboardFilterGroup: React.FC<Props> = ({
     ))}
   </div>
 );
+
+// hover:bg-indigo-700
```
Comment (brettvniekerk) [inline src/components/Dashboard/DashboardFilterGroup.tsx:42]: remove comment please

File: src/components/Toggle.tsx [hunk 3]
```diff
@@ -9,7 +9,7 @@ export const Toggle: React.FC<Props> = ({ enabled, onChange }) => (
   <Switch
     checked={enabled}
     onChange={onChange}
-    className={`${enabled ? "bg-blue-600" : "bg-gray-200"}
+    className={`${enabled ? "bg-indigo-600" : "bg-gray-200"}
```
Comment (brettvniekerk) [inline src/components/Toggle.tsx:12]:
```ts
className={classnames({
    "bg-indigo-600": enabled,
    "bg-gray-200": !enabled
})}
```

**Review by brettvniekerk — CHANGES_REQUESTED** [review]
LGTM but let's make sure we are consistent and use the `classnames` library for reactive styling please

**Review by brettvniekerk — APPROVED** [review]

**Review by OptimusOpus — APPROVED** [review]

**Pull request merged**